MetaMask/metamask-mobile
# fix: use correct import for MetricsEventBuilder

## **Description**



Fixes correct import for MetricsEventBuilder


## **Related issues**

Fixes: [12530](https://github.com/MetaMask/metamask-mobile/issues/12530)

[Related Slack message](https://consensys.slack.com/archives/C8RSKCNCD/p1734622035414719?thread_ts=1734535680.397999&cid=C8RSKCNCD)

## **Manual testing steps**

1. Go to this [dapp ](https://metamask.github.io/metamask-sdk/chore-update-vuejs-build/packages/examples/react-demo/build/index.html)in the inapp browser with a debug build
2. Check the console logs for the dapp
3. Switch chains -> See console

## **Screenshots/Recordings**
[Link to zoom recording](https://consensys.zoom.us/rec/share/AJr-gv8tqRuEZx0PorROKgux41aYESIZ3k_PHRMoYTwh6EHIcZU40oliXHa28bYU.wYfZYWsUupmI5hQo)


### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
